docs/user-guide/projects.md

```diff
@@ -54,6 +54,75 @@ argocd proj add-destination <PROJECT> <CLUSTER>,<NAMESPACE>
 argocd proj remove-destination <PROJECT> <CLUSTER>,<NAMESPACE>
 ```
 
+We can also do negations of destinations (i.e. install anywhere _apart from_).
+
+```bash
+argocd proj add-destination <PROJECT> !<CLUSTER>,!<NAMESPACE>
+argocd proj remove-destination <PROJECT> !<CLUSTER>,!<NAMESPACE>
+```
+
+Declaratively we could do something like this:
+
+```yaml
+spec:
+  destinations:
+  # Do not allow any app to be installed in `kube-system`  
+  - namespace: '!kube-system'
+    server: '*'
+  # Or any cluster that has a URL of `team1-*`   
+  - namespace: '*'
+    server: '!https://team1-*'
+    # Any other namespace or server is fine though.
+  - namespace: '*'
+    server: '*'
+```
+
+Doing this is order-dependent though, and this could also lead to unexpected behavior. Take this example:
```

Comment:
I don't think I'd call it "order-dependent". I'd say "By default, if _any_ destination matches, then the resource is allowed. When using negation, this default behavior can cause a `destinations` field that _looks_ correct to allow resources that you actually want to block. Take this example:"